### DISCUSSION TITLE
Welcome to hacktoberfest2021 Discussions!

### DISCUSSION THREAD

**Discussion by abhpd** [abhpd/hacktoberfest2021#321] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arvindkr7** [comment]
I want to contribute as python projects that would be mostly related to AI and ML. But, there are limited project directories inside Python. Can I add a new directory for my project?
